.travis.yml

```diff
@@ -529,4 +620,7 @@ jobs:
     - stage: Trigger deb and rpm package build (nightly release)
       name: Trigger deb and rpm package build
       script: .travis/trigger_package_generation.sh "[Build latest]"
-      after_failure: post_message "TRAVIS_MESSAGE" "<!here> Failed to trigger deb and rpm package build during nightly release" "${NOTIF_CHANNEL}"
+      after_failure: >
+        post_message "TRAVIS_MESSAGE"
+        "<!here> Failed to trigger deb and rpm package build during nightly release"
+        "${NOTIF_CHANNEL}"
```

Comment:
```
diff -urN <(cat .travis.yml | yq .) <(git show fb072738d387dd7623b6f995a86b694f8cca37ac:.travis.yml | yq .)
```
:point_up: indicates that the before and after `.travis.yml`s are not equivalent